research/deep_speech/deep_speech.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Deep Speech
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=g-bad-import-order
+import numpy as np
+from absl import app as absl_app
+from absl import flags
+import tensorflow as tf
+# pylint: enable=g-bad-import-order
+
+import deep_speech_model
+import data.dataset as dataset
+from official.utils.export import export
+from official.utils.flags import core as flags_core
+from official.utils.logs import hooks_helper
+from official.utils.logs import logger
+from official.utils.misc import distribution_utils
+from official.utils.misc import model_helpers
+
+
+def convert_keras_to_estimator(keras_model, num_gpus, flags_obj):
+  """Configure and convert keras model to Estimator.
+  Args:
+    keras_model: A Keras model object.
+    num_gpus: An integer, the number of gpus.
+    flags_obj: An object containing parsed flags. See define_resnet_flags()
+      for details.
+  Returns:
```

Comment:
Nit: (Unsure if the linter complains about this) Might need an extra line between the end of Args and Returns.